packages/data-point-service/README.md

```diff
@@ -161,6 +166,27 @@ DataPointService.create({
 })
 ```
 
+### cache.revalidateTimeout
+
+```js
+staleWhileRevalidate: String|Number
+```
```

Comment:
please list the default value up here instead of at the end of the paragraph. something like this:

```js
staleWhileRevalidate: String|Number

default: '5s'
```